demo-pipeline/buildspec/instance-backup.yaml

```diff
@@ -0,0 +1,30 @@
+version: 0.2
+
+env:
+  variables:
+    ELASTIO_IMAGE: public.ecr.aws/elastio-dev/elastio-cli:0.9.0-master-amd64
```

Comment:
This is a very old version of the CLI.  The current release is 0.13.0.  You should never use the `master` version because that's built directly from our `master` branch and often is broken.  You should use `0.13.0-amd64` for now.  We have a task to make `latest` correspond to the most recent release, but right now `latest` is the most recent build regardless of what branch, which means it's highly unstable and should not be used yet.

Also note that, starting in 0.14.0, you won't need to append `-amd64` because we publish multi-arch binaries that support both amd64 and aarch64 architectures.

Reply:
Removed this altogether since it is unneeded.